service/src/main/java/bio/terra/pipelines/stairway/imputation/SubmitCromwellRunSetStep.java

```diff
@@ -63,45 +66,179 @@ public StepResult doStep(FlightContext flightContext)
               "couldn't find method version id for method name " + wdlMethodName));
     }
 
-    // launch a cbas submission
+    // input definitions - hardcoded for now
+    // in future this will be pulled from the workspace
+    String workspaceStorageContainerUri =
```

Comment:
hmm should we make a new ticket to add workspace storage container to the pipelines table / an admin endpoint to update like the control workspace id?

Reply:
yeah not sure exactly how it should work (another workspace id? just the storage container url? it's so custom) but i can make a ticket and we can refine

Reply:
https://broadworkbench.atlassian.net/browse/TSPS-242